js/app.js

```diff
@@ -58,6 +59,21 @@ var App = function() {
 		}
 	};
 
+	var checkDeprecatedOptions = function(userConfig) {
+		var deprecatedOptions = require(__dirname + "/../config/deprecated.js");
```

Comment:
I thought about it. But since for now _deprecated_ is limited purely to config options, and the main _js_ folder is quite full as it is, I decided to put the file into the config directory. It can always be moved later, if more than just deprecated config options are added or other reasons arise. I am open to discussion on this, of course.